src/macrofier/engine.mjs

```diff
@@ -1089,9 +1095,10 @@ function insertMethodMacros(template, methodObj, json, templates, examples = {})
   const serializedParams = types.getSchemaInstantiation(methodObj, json, methodObj.name, { instantiationType: 'params' })
   const resultInst = types.getSchemaInstantiation(result.schema, json, result.name, { instantiationType: 'result' } )
   const serializedEventParams = event ? indent(types.getSchemaInstantiation(event, json, event.name, { instantiationType: 'params' }), '    ') : ''
-  const callbackSerializedParams = event ? types.getSchemaInstantiation(event, json, event.name, { instantiationType: 'callback.params' }) : ''
-  const callbackResultInst = event ? types.getSchemaInstantiation(event, json, event.name, { instantiationType: 'callback.result' }) : ''
-  const callbackResponseInst = event ? types.getSchemaInstantiation(event, json, event.name, { instantiationType: 'callback.response' }) : ''
+  const callbackSerializedParams = event ? types.getSchemaInstantiation(event, json, event.name, { instantiationType: 'callback.params', prefix: method.alternative }) : ''
```

Comment:
method.alternative is for suggesting a new method to use instead of a deprecated method... not sure why you are using it here.

Reply:
it is used to get proper method name referring this property. say if it is onNetworkChanged, the alternative name is 'network'. the generate name of the component schema for networkInfo, is now NetworkNetworkInfo. but the name in the result scheme is still netowrkInfo. So used this alternative name to populate it

Reply:
oops, yes, you are correct.